CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
 
+## Unreleased
+
+  * **Internal**
+
+    * `babel-types`: Start test suite.
+
```

Comment:
Oh, I don't know -- not familiar with it either. Good question though. A quick search turns up some stuff that seems to be discussing that very thing. I'll have to learn more about it sometime.

* https://github.com/isaacs/github/issues/487
* https://github.com/olivierlacan/keep-a-changelog/issues/56
* https://about.gitlab.com/2015/02/10/gitlab-reduced-merge-conflicts-by-90-percent-with-changelog-placeholders/